docs/guides/tanzu-mission-control_gitiops.md

```diff
@@ -151,7 +151,7 @@ resource "tanzu-mission-control_git_repository" "create_cluster_git_repository"
 
 # Create cluster level Kustomization
 resource "tanzu-mission-control_kustomization" "create_cluster_kustomization" {
-  name = "tf-kustomizationy-name" # Required
```

Comment:
I hope you have generated these docs using plugin doc and not edited manually?

Reply:
yes
I've generated them